### DISCUSSION TITLE
Access Postal from Caddy that is not running on same machine

### DISCUSSION THREAD

**Discussion by pahlgrimm** [postalserver/postal#3209] [Help with using Postal]
Hi, I am trying to get Postal running. It runs but the web container listens only on 127.0.0.1:5000.
I have a Caddy running on an opnsense appliance. It seems that Postal ist ignoring everything coming from the router/gateway.

**Comment by JohnHammell** [comment]
I believe you may need to run two Caddy installs for this. The local Caddy which proxies connections to Postal in the local environment. Then the Caddy install on the different machine. You end up with: Caddy (external) -> Caddy (local to Postal) -> Postal.

This is how I have things setup, however I use HAProxy instead of Caddy. So I have one container running HAProxy that receives inbound http/s traffic, which proxies to HAProxy in my Postal container which then proxies to Postal. HAProxy -> HAProxy -> Postal.

**Reply by pahlgrimm** [reply]
I finally got it working! I just added the following entries to /opt/postal/config/postal.yml:
```
postal:
  trusted_proxies: [IP of my Caddy]
web_server:  
  default_bind_address: IP of host
```
restarted it per `postal restart` and it works like a charm :-D
